apps/web/app/(auth-pages)/sign-in/page.tsx

```
@@ -97,21 +96,33 @@
 						</div>
 					)}
 
 					{authError && !isNotRegistered && (
-						<div className="text-red-600">
+						<div className="text-red-600" role="alert" aria-live="assertive" tabIndex={0}>
 							There was an error during authentication. Please try again.
 						</div>
 					)}
+
+					{authSuccess && (
+						<output className="text-green-600" aria-live="polite" tabIndex={0}>
```

Biome (1.9.4)

[error] The HTML element output is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)